### ISSUE TITLE
LOC authorities not resolving

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/openvault3#554]
On a record with a transcript, LOC authorities are not resolving. See this record: http://openvault.wgbh.org/catalog/A_739CF821AA9F44B891160EB2CF76FA78. 

Click on "Congress" or "NAACP" for an example.

**Comment by mccalluc** [comment]
@caseyedavis12, see also: #422. Not that name authority files don't rock, but it's not necessarily the most useful resource for an end user.

Proposal: Change all of these to point to wikipedia entries. There are other possibilities, but I'll start downloading, and getting a tool-set together, and at some point we can figure out what the best solution is.

**Comment by mccalluc** [comment]
Proposal: Where we have TEI like
```xml
<keywords scheme="http://authorities.loc.gov">
  <term 
    xml:id="March_on_Washington" 
    xhtml:href="http://authorities.loc.gov/cgi-bin/Pwebrecon.cgi?AuthRecID=4731532&amp;v1=1&amp;HC=1&amp;SEQ=20110607152950&amp;PID=3SzQ1vk4WXbi5PnkhD9RElmIzeFLyo">
      March on Washington for Jobs and Freedom, Washington, D.C., 1963
  </term>
  ...
```
replace it with 
```xml
<keywords scheme="http://authorities.loc.gov">
  <term 
    xml:id="March_on_Washington" 
    xhtml:href="https://en.wikipedia.org/wiki/Washington,_D.C."
    target="http://authorities.loc.gov/cgi-bin/Pwebrecon.cgi?AuthRecID=4731532&amp;v1=1&amp;HC=1&amp;SEQ=20110607152950&amp;PID=3SzQ1vk4WXbi5PnkhD9RElmIzeFLyo">
      March on Washington for Jobs and Freedom, Washington, D.C., 1963
  </term>
  ...
```

Where the target attribute comes from the TEI schema: http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.pointing.html#tei_att.target

- Ok? There are other ways it could be done.
- Also, should we be favoring wikipedia everywhere? Are there any sources we would like to preserve?

**Comment by Muraszko** [comment]
@mccalluc @caseyedavis12 
I definitely want to sit down and talk about this a little more before we make any decisions.

**Comment by caseyedavis12** [comment]
@muraszko +1. I don't have much institutional knowledge when it comes to the Open Vault transcripts and grant requirements, and I'm wondering why we don't just hyperlink these terms as keywords to search other content in the collection like we do with the other fields in the metadata records.

**Comment by sroosa** [comment]
@mccalluc and I were talking about this yesterday. We noticed a few things. 
1. This is something that is in the older Open Vault transcripts (Vietnam and a few Nuclear Age), but isn't in all of the ones we put up later. Which to me means that we should keep it where we have it, and not worry about adding it to the others yet, but possibly plan on doing that in the future/setting up guidelines for doing it in general.
2. The authoritative sources aren't bad, they are just not the most useful things to send our users to. I agree that wikipedia is probably the best choice for that, but I don't think we should just throw the authorities away. Chuck and I discussed moving the authority urls to a different attribute in the TEI XML, and using more useful urls for the hrefs. 
3. I just wanted to say as someone who worked on a few of the transcripts, it's awesome that we can link out from within the body of the transcript!
